code/src/hardware_usart.cpp

```diff
@@ -3,128 +3,136 @@
 
 namespace r2d2 {
 
-    bool hardware_usart_c::transmit_ready() {
-        return (hardware_usart->US_CSR & 2);
-    }
+bool hardware_usart_c::transmit_ready() { return (hardware_usart->US_CSR & 2); }
 
-    void hardware_usart_c::send_byte(const uint8_t &b) {
-        while (!transmit_ready()) {
-            hardware_usart->US_THR = b;
-        }
+void hardware_usart_c::send_byte(const uint8_t &b)
+{
+    while (!transmit_ready())
```

Comment:
it should be `while (!transmit_ready()){}`
or:
`while (!transmit_ready()){

}`